### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by Mmalumbo**

### CODE REVIEW

File: src/models/MigrationDataElements.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const fields: DefineModelAttributes<object> = {
   dataElementCode: STRING,
   organizationUnitCode: STRING,
   isProcessed: BOOLEAN,
-  migratedAt: STRING,
+  migratedAt: Date,
```
Comment (liwoo) [inline src/models/MigrationDataElements.ts]: should be consistent with Sequilize Types

File: src/migrate/helpers.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const createChunkCounter = async (
   chunkSize: number
 ): Promise<number[]> => {
   const MigrationDataElement = await createMigrationDataElements(sequelize);
-  const where = { migrationId, isProcessed: false };
+  const where = { migrationId, migratedAt: null };
```
Comment (liwoo) [inline src/migrate/helpers.ts]: Business Logic is that this worker is only taking things that are NOT processed, while Failure Worker takes whats NOT migrates but IS processed

File: src/migrate/migrate.ts [hunk 3]
```diff
@@ -91,6 +79,11 @@ export const migrate = async (
 
     await pushToFailQueue(worker, message);
   } else {
+    await updateMigrationDataElements(
```
Comment (liwoo) [inline src/migrate/migrate.ts]: must be a helper function called `persistFailedMigrationIDs` etc

File: src/models/MigrationDataElements.ts (hunk 1 above)
Comment (Mmalumbo) [reply src/models/MigrationDataElements.ts]: Changed date to use sequelize DATE type

**Comment by Mmalumbo** [comment]
made the changes, you can check the code base again.

File: src/migrate/migrate.ts [hunk 4]
```diff
@@ -71,15 +73,15 @@ export const migrate = async (
     offset++;
   }
 
-  await updateMigrationDataElements(
+  await persistSuccessfulMigrationDataElements(
     sequelize,
```
Comment (liwoo) [inline src/migrate/migrate.ts]: change sequilize to connection

**Pull request merged**